server/controllers/rpaFrameworkCommandsController.js

```diff
@@ -33,27 +34,37 @@ exports.getAvailableTasksForApplications = async (req, res) => {
 // GET /rpa-framework/commands/get-vars-for-task?application=Browser&task=Open+Browser
 exports.getVarsForTask = async (req, res) => {
   try {
-    const {application} = req.query;
-    const {task} = req.query;
+    const { application } = req.query;
+    const { task } = req.query;
     res.set('Content-Type', 'application/json');
 
     if (application != null && task != null) {
       mongoose.model('rpa-task').findOne(
-      {
-        Application: application,
-        Task: task,
-      }, 
-      {
-        InputVars: 1,
-        Output: 1
-      }, 
-      (err, tasks) => {
-        res.send(tasks);
-      });
+        {
+          Application: application,
+          Task: task,
+        },
+        {
+          InputVars: 1,
+          Output: 1
+        },
+        (err, tasks) => {
+          res.send(tasks);
+        });
```

Comment:
please remove all these changes (all in this file above here), as they were not necessary. I know it picky, but I will get a lot of merge conflicts because of this